### PR TITLE
Euf 9 create crud service with repo basket

### PR DESCRIPTION
**Pull request by Jamshidx8ja**

### CODE REVIEW

File: src/main/java/uz/pdp/eufloria/controller/CategoryController.java [hunk 1]
```diff
@@ -55,3 +55,17 @@ public ResponseEntity<?> removeCategoryFromProduct(@PathVariable @NotBlank Strin
...
+
+
+
+
+
+
+
+
+
+
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/controller/CategoryController.java]: Remove empty lines

File: src/main/java/uz/pdp/eufloria/domain/Basket.java [hunk 2]
```diff
@@ -1,26 +1,20 @@
...
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Id;
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
+import jakarta.persistence.*;
+import lombok.*;
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/domain/Basket.java]: remove wild card

File: src/main/java/uz/pdp/eufloria/service/BasketService.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import uz.pdp.eufloria.dto.response.BasketDto;
+
+import java.util.List;
+
+@Service
+public interface BasketService {
+//    BasketDto save(BasketDto basketDto);
+//    BasketDto update(BasketDto basketDto);
+//    void delete(Long id);
+    BasketDto getById(Long id);
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/service/BasketService.java:9]: remove commented part

File: src/main/java/uz/pdp/eufloria/service/impl/BasketServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+//
+//    }
+//
+//    @Override
+//    public void delete(Long id) {
+//        if(!basketRepository.existsById(id))
+//            throw new NotFoundException("Basket");
+//        basketRepository.deleteById(id);
+//    }
+
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/service/impl/BasketServiceImpl.java]: remove unused mehtods

File: src/main/java/uz/pdp/eufloria/controller/BasketController.java [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (byte-11) [inline src/main/java/uz/pdp/eufloria/controller/BasketController.java]: remove empty lines

**Review by ismoil0709 — APPROVED** [review]

**Pull request merged**